package.json

```diff
@@ -50,6 +50,7 @@
     "sinon": "^1.17.3",
     "travis-after-all": "^1.4.4",
     "validate-commit-msg": "2.6.1",
+    "which": "^1.2.8",
```

Comment:
Just noticed this was added as a dependency in this PR but it's not being used in this PR. Was this a mistake?